sdks/python/apache_beam/transforms/core.py

```diff
@@ -875,18 +875,20 @@ class CombineFn(WithTypeHints, HasDisplayData, urns.RunnerApiFn):
   input argument, which is an instance of CombineFnProcessContext). The
   combining process proceeds as follows:
 
-  1. Input values are partitioned into one or more batches.
-  2. For each batch, the create_accumulator method is invoked to create a fresh
+  1. The setup method is invoked.
```

Comment:
This alludes that  setup method is called once per entire collection. Wouldn't it be called per batch? Aggregation may happen on multiple workers, and I imagine that in such case each worker will call setup/teardown methods. Should we switch step 1 and 2?

Reply:
It probably should be like this:

```
1. Input values are partitioned into one or more batches.
2. For each batch, the setup method is invoked.
3. For each batch, the create_accumulator method is invoked...
```